lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -1364,6 +1364,13 @@ void request_evaluate(const llvm::json::Object &request) {
   std::string expression = GetString(arguments, "expression").str();
   llvm::StringRef context = GetString(arguments, "context");
 
+  // Remember the last non-empty expression from the user, and use that if
+  // the current expression is empty (i.e. the user hit plain 'return').
+  if (!expression.empty())
+    g_dap.last_nonempty_expression = expression;
+  else
```

Comment:
Should this be done using https://github.com/llvm/llvm-project/blob/ea2da571c761066542f8d2273933d2523279e631/lldb/include/lldb/API/SBCommandInterpreterRunOptions.h#L77-L81 on the https://github.com/llvm/llvm-project/blob/ea2da571c761066542f8d2273933d2523279e631/lldb/tools/lldb-dap/LLDBUtils.cpp#L48C14-L48C27 call?

Reply:
The SBCommandIntepreterRunOptions don't seem to be used on this execution path, so I couldn't make use of this.